provider/resource_site_alert.go

```diff
@@ -33,11 +35,23 @@ func resourceSiteAlert() *schema.Resource {
 				Type:        schema.TypeInt,
 				Description: "The number of minutes of past traffic to examine. Must be 1, 10 or 60.",
 				Required:    true,
+				ValidateFunc: func(val interface{}, key string) (warns []string, errs []error) {
```

Comment:
[NITS] It may not be a good idea to write `ValidateFunc` in the resource file, as it will reduce readbility 🤔 

For now, I wrote it the same as the existing one.
https://github.com/signalsciences/terraform-provider-sigsci/blob/c9d0ad84fe0a3c6f04a9add51915f25a8f82269a/provider/resource_corp_cloudwaf_instance.go#L42-L50
https://github.com/signalsciences/terraform-provider-sigsci/blob/c9d0ad84fe0a3c6f04a9add51915f25a8f82269a/provider/resource_corp_rule.go#L30-L35

Reply:
This is fine since like you said it matches the existing code 👍